src/audio/eq_iir/eq_iir.c

```diff
@@ -271,7 +271,7 @@ SOF_MODULE_INIT(eq_iir, sys_comp_module_eq_iir_interface_init);
 SOF_LLEXT_MOD_ENTRY(eq_iir, &eq_iir_interface);
 
 static const struct sof_man_module_manifest mod_manifest __section(".module") __used =
-	SOF_LLEXT_MODULE_MANIFEST("EQIIR", eq_iir_llext_entry, 1, UUID_EQIIR);
+	SOF_LLEXT_MODULE_MANIFEST("EQIIR", eq_iir_llext_entry, 1, UUID_EQIIR, 40);
```

Comment:
Can we do a -1 i.e. dont care ?

Reply:
@lgirdwood actually we do care now. Not because LLEXT modules need this, but because it's used by the infrastructure. We should remove this for LLEXT, but that should be coordinated with other loadable module implementations

Reply:
ok, lets circle back on this, we need to avoid artificial hard coded limitations

Reply:
@lgirdwood this is where it's used and this is where it used to fail without these two commits: https://github.com/thesofproject/linux/blob/8245a1411fef45a208f182c176660107b7f582aa/sound/soc/sof/ipc4-topology.c#L1119 So we'd need to modify the driver to allow arbitrarily large instance IDs, or we can define and use a macro like `#define LLEXT_MODULE_INSTANCE_COUNT 65535` and use it for all LLEXT modules?

Reply:
@lyakh pls align with @ujfalusi re driver updates as he's back soon.

Reply:
we can as well use `ida_alloc()` in kernel and ignore the max, but I don't know what `LLEXT` vs `non LLEXT` modules are and why would they need to be handled differently...